### ISSUE TITLE
Check Row Count Validation - V1.0.12 

### ISSUE THREAD

**Issue by leejaykar** [unchartedsoftware/PLUTO#559]
change row count logic to < user entered number for Minimum Number...

For number of rows n in table fails:
Minimum Number of Data Rows Otherwise Error n
Maximum Number of Data Rows Otherwise Error n

This is the current way to use this validation and it is confusing:
Minimum Number of Data Rows Otherwise Error n-1
Maximum Number of Data Rows Otherwise Error n

**Comment by pandersen-uncharted** [comment]
The logic is as you would like (i.e. `<` rather than `<=`) however I noticed the code doesn't ignore header rows, and I think that's where the confusion comes in. I can change it so that it only counts data rows.

**Comment by pandersen-uncharted** [comment]
The code does use `<` rather than `<=`. i.e
```javascript
if (rowCount < minErrorThreshold)
	error(`Number of rows (${rowCount}) less than error threshold (${minErrorThreshold}).`);
else if (rowCount < minWarningThreshold)
	warning(`Number of rows (${rowCount}) less than warning threshold (${minWarningThreshold}).`);
```

It skips the header rows so could that be the problem? The row count is for data rows rather than all rows in the file.

**Comment by pandersen-uncharted** [comment]
Allow the min and max to be the same so it's possible to check for files of an exact length.

**Comment by leejaykar** [comment]
"Allow the min and max to be the same so it's possible to check for files of an exact length." 
This is the outcome I'd like.  I see how the logic is set up as defining the threshold, maybe if we adjust the UI wording it would be better without having to change the logic.

**Comment by pandersen-uncharted** [comment]
This will be available in 1.0.13.
